{"name": "修改他人用户信息，修改失败提示无权限操作", "status": "failed", "statusDetails": {"message": "Failed: ScopeMismatch: You tried to access the 'module' scoped fixture 'creat_data' with a 'session' scoped request object, involved factories\ncase\\conftest.py:23:  def login(creat_data)\ncase\\conftest.py:11:  def creat_data()", "trace": "ScopeMismatch: You tried to access the 'module' scoped fixture 'creat_data' with a 'session' scoped request object, involved factories\ncase\\conftest.py:23:  def login(creat_data)\ncase\\conftest.py:11:  def creat_data()"}, "start": 1585792749300, "stop": 1585792749300, "uuid": "b723e79c-a828-420e-b077-b8d3ab5aa712", "historyId": "0a871938a144a3a54f502d8d89dc85bb", "testCaseId": "f065d1a08693963565b0cd7dcd4366c7", "fullName": "case.updateInfo_case.test_up_userInfo#test_up_userInfo_02", "labels": [{"name": "story", "value": "修改个人信息接口测试"}, {"name": "parentSuite", "value": "case.updateInfo_case"}, {"name": "suite", "value": "test_up_userInfo"}, {"name": "host", "value": "goushuai"}, {"name": "thread", "value": "21292-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.updateInfo_case.test_up_userInfo"}]}